dist/gmusic.js

```diff
@@ -239,6 +245,87 @@ proto.playback = {
   }
 };
 
+// Create a queue API
+proto.queue = {
+  _fireEvent: function (el, etype) {
+    if (el.fireEvent) {
+      el.fireEvent('on' + etype);
```

Comment:
Fair enough, I just copy pasted that code from another project which required intereting levels of IE support.  It looks like chrome doesn't support `el.fireEvent` anyway so I'll just kill it :+1: